docker-compose.yml

```diff
@@ -25,12 +25,11 @@ hydroshare:
     SOLR_HOST: solr
   volumes:
     # hydroshare repository
-    - ".:/home/docker/hydroshare"
-    #- "/home/hydro/hydroshare:/home/docker/hydroshare"
+    - "/home/hydro/hydroshare:/home/docker/hydroshare"
     # pycharm debugging
     # - "/home/hydro/pycharm-debug:/home/docker/pycharm-debug"
     # log files
-    #- "/home/hydro/hydroshare/log:/var/log/hydroshare"
```

Comment:
Please restore original docker-compose.yml from the current develop branch. This file should not be checked into your branch since it is changed dynamically by hsctl script build according to docker-compose.template. Changing of this default docker-compose.yml may break hsctl build logic, at least I know for sure this file should not be changed, especially with a hardcode path containing hydro.